pom.xml

```diff
@@ -109,21 +110,20 @@
             <artifactId>springfox-swagger-ui</artifactId>
             <version>2.6.1</version>
         </dependency>
-        <!-- <dependency>
+        <dependency>
             <groupId>javax.el</groupId>
             <artifactId>javax.el-api</artifactId>
-            <version>3.0.0</version>
-        </dependency> -->
+            <version>2.2.4</version>
+        </dependency>
         <dependency>
-            <groupId>org.glassfish</groupId>
+            <groupId>org.glassfish.web</groupId>
             <artifactId>javax.el</artifactId>
-            <!-- version>2.2.4</version> -->
-            <version>3.0.1-b08</version>
+            <version>2.2.4</version>
```

Comment:
Why it's downgrading again javax.el?

Reply:
Looks like javax.el should not be there and the glassfish dependency was a bad merge on my part.  The develop branch currently has:

```
        <!-- <dependency>
            <groupId>javax.el</groupId>
            <artifactId>javax.el-api</artifactId>
            <version>3.0.0</version>
        </dependency> -->
        <dependency>
            <groupId>org.glassfish</groupId>
            <artifactId>javax.el</artifactId>
            <!-- version>2.2.4</version> -->
            <version>3.0.1-b08</version>
        </dependency>
```
I'll fix that; good catch.